electron/windows/web3authSwapOwner.js

```diff
@@ -0,0 +1,115 @@
+const { BrowserWindow } = require('electron');
+const path = require('path');
+
+const { logger } = require('../logger');
+
+/** @type {Electron.BrowserWindow | null} */
+let web3AuthSwapOwnerWindow = null;
+const getWeb3AuthOwnerSwapWindow = () => web3AuthSwapOwnerWindow;
+
+/**
+ * Create the web3auth window for displaying web3auth modal for swapping owner
+ * @param {string} baseUrl - Base URL for the window
+ * @param {object} params - Transaction parameters
+ */
+/**
+ * Creates the Web3Auth Swap Owner window.
+ * @param {string} baseUrl - Base URL for the window.
+ * @param {{safeAddress: string, oldOwnerAddress: string, newOwnerAddress: string, backupOwnerAddress: string, chainId: number}} params - Transaction parameters.
+ * @returns {Promise<Electron.BrowserWindow|undefined>} The created BrowserWindow instance or undefined.
+ */
+const createWeb3AuthSwapOwnerWindow = async (baseUrl, params) => {
+  if (
+    !getWeb3AuthOwnerSwapWindow() ||
+    getWeb3AuthOwnerSwapWindow().isDestroyed
+  ) {
+    web3AuthSwapOwnerWindow = new BrowserWindow({
+      title: 'Web3Auth Swap Owner',
+      resizable: false,
+      draggable: true,
+      frame: false,
+      transparent: true,
+      fullscreenable: false,
+      maximizable: false,
+      closable: false,
+      width: 480,
+      height: 700,
```

Comment:
should we reuse them from a variable, i see them being used at a few other places.